server/tests/unit/test_sudoku.py

```diff
@@ -135,3 +138,21 @@ def test_sudoku_object_correct_input() -> None:
     assert S.is_solvable
     hint = S.hint
     assert VALID_SUDOKU_HARD_SOLVED[hint[0]] == hint[1]
+
+
+def test_verify_solution() -> None:
```

Comment:
Jag hade nog separerat ut dessa till två test cases, en för verify och en för not verify.
```python
test_can_verify_solution() -> None:
    ...

test_cannot_verify_solution() -> None:
    ...
```